generate-third-party-notices.sh

```diff
@@ -0,0 +1,9 @@
+PACKAGE_DIR=$(pwd)
```

Comment:
That file is the one that Yen provided that she used to deal with generating notices for monorepos.  In the past, we've had one `package.json` at the root level of the project with its `package.json` and we could easily just run `generate-license-file` to generate the `THIRD PARTY NOTICES` file.  Since there are multiple `package.json` files in subdirectories, we need a slightly different approach.  If we just run `generate-license-file` in the subdirectories, since they don't have access to the `package-lock.json` file, will generate notices for the direct dependencies, but not the transitive dependencies. This script takes the files in the subdirectories and brings them to a level where they can access the `package-lock.json` file so that notices for transitive dependencies can be generated.  I am slightly unclear about how the very last part of that works, but I will ask Yen and get back to you on that.

Reply:
I talked with Yen and might try one more approach where I edit Workspaces in the top level `package.json` so we can avoid the copying and pasting stuff.

Reply:
I made a new approach where instead of moving the contents of `studio` and `studio-plugin` to a file outside of the workspace, I just temporarily remove `studio` and `studio-plugin` from the workspace, and re-add them once I finish making the third party notices.